crm_win_reason/models/crm_lost_reason.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class CrmLostReason(models.Model):
+    _inherit = "crm.lost.reason"
+
+    reason_type = fields.Selection(
+        [("won", "Won"), ("lost", "Lost")],
```

Comment:
Should we have a default value ? I am thinking of already existing records especially (set as `lost`)